### DISCUSSION TITLE
Failed to create mod instance  ModID: computercraft, class dan200.computercraft.ComputerCraft

### DISCUSSION THREAD

**Discussion by samuelh2005** [cc-tweaked/CC-Tweaked#846] [Q&A] [answered]
### Minecraft Version

1.16.x

### Version

1.97.0

### Details

So i've been making an [addon mod](https://github.com/ajh123-development/better-cc) for cc. It was working before i've updated to forge 36.1.32 but now i get this [error](https://github.com/ajh123-development/static-stuff/blame/main/logs/better-cc/cc-t%7Cmod%20instance%20error%20forge:1.16.5-36.1.32.md#L358) when i compile my mod that uses the official mappings, this part of the error may be to do with it ```java.lang.NoSuchMethodError: net.minecraft.util.ResourceLocation.func_110623_a()Ljava/lang/String;```

**Comment by SquidDev** [comment, marked as answer]
Having a look at your build.gradle file, it looks like you're not using `fg.deobf` ([see the readme](https://github.com/SquidDev-CC/CC-Tweaked#using)), hence the mod isn't being deobfuscated.

**Reply by samuelh2005** [reply]
Thanks i will try it!

**Reply by samuelh2005** [reply]
Works now, yaaa!!!
